examples/quick-start/bentoml-quick-start-guide.ipynb

```diff
@@ -77,7 +77,7 @@
     "@env(conda_pip_dependencies=[\"scikit-learn\"])\n",
     "class IrisClassifier(BentoService):\n",
     "\n",
-    "    @api(DataframeHandler)\n",
+    "    @api(DataframeHandler, input_dtypes={\"col1\":\"float64\",\"col2\":\"float64\",\"col3\":\"float64\",\"col4\":\"float64\"})\n",
```

Comment:
maybe remove it from the quick_start guide for now, until we iterate on this a bit more?